app_dart/lib/src/service/scheduler.dart

```diff
@@ -466,20 +465,28 @@ class Scheduler {
             final String checkName = checkRunEvent.checkRun!.name!;
             final RepositorySlug slug = checkRunEvent.repository!.slug();
 
-            // TODO: Can't access branch from [checkRunEvent.checkRun.checkSuite] because head_branch is not deserialized
+            // TODO(nehalvpatel): Can't access branch from [checkRunEvent.checkRun.checkSuite] because head_branch is not deserialized
```

Comment:
Flutter TODO style follows the format:

TODO(username): Sentence summary. https://github.com/flutter/flutter/issue/#####